sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_promptflow/azure/_lite_azure_management_client.py

```diff
@@ -0,0 +1,201 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from logging import Logger
+from typing import Any, Dict, Final, Optional, Set, Union, cast
+from threading import Lock
+from urllib.parse import quote
+from json.decoder import JSONDecodeError
+
+from azure.core.credentials import TokenCredential, AzureNamedKeyCredential, AzureSasCredential
+from azure.core.rest import HttpResponse
+from azure.ai.evaluation._exceptions import ErrorBlame, ErrorCategory, ErrorTarget, EvaluationException
+from azure.ai.evaluation._http_utils import HttpPipeline, get_http_client
+from azure.ai.evaluation._promptflow.azure._token_manager import AzureManagementAPITokenManager
+from azure.ai.evaluation.simulator._model_tools._identity_manager import TokenScope
+from ._models import BlobStoreInfo, Workspace
+
+
+API_VERSION: Final[str] = "2024-10-01"
+QUERY_KEY_API_VERSION: Final[str] = "api-version"
+PATH_ML_WORKSPACES = ("providers", "Microsoft.MachineLearningServices", "workspaces")
+
+
+class LiteAzureManagementClient:
```

Comment:
Nit: Calling it **LiteAzureManagementClient** might be too generic. How about naming it **LiteMLClient** or simply **WorkspaceClient** instead?